### PR TITLE
Telemetry input mct

### LINKED ISSUE #570: All subsystem telemetry input into MCT
### Need to input the remaining subsystems of telemetry into OpenMCT for GSW

**Four steps for each sybsystem/domain object:**

- Generate subsystem json file
- input instillation config in generic-plugin
- install using the generic-plugin in index
- add initial telemetry values into state-variables

### PR DESCRIPTION
**Pull request by Duncan-McD**
# Automatic MCT Configuration and MCT telemetry subsystem
Fixes #570 

### Summary of changes
 - Added generators for automatically parsing telemetry.txt and flow_data.cpp into MCT configurations
 - Ran generators to get updated MCT configurations

If it applies, take a chance to describe offshoot work or TODOs that are a part of your PR.

### Testing
Ran using various PTests to see data in Open MCT
Note: to run and test with Elastic Search IMEI number in json config must also match the index name on line 16 of MCT/server-files/telemetry.js

### Telemetry

Telemetry has been setup to port into MCT

### Documentation Evidence
Documentation is added into the python generator files to explain the thought process and generation

### CODE REVIEW

File: MCT/public/index.html [hunk 1]
```diff
@@ -26,6 +26,26 @@
...
+        openmct.install(GenericPlugin("attitude_estimator"));
+        openmct.install(GenericPlugin("attitude"));
+        openmct.install(GenericPlugin("adcs_monitor"));
+        openmct.install(GenericPlugin("prop"));
+        openmct.install(GenericPlugin("adcs_command"));
+        openmct.install(GenericPlugin("adcs"));
+        openmct.install(GenericPlugin("radio"));
+        openmct.install(GenericPlugin("docksys"));
+        openmct.install(GenericPlugin("dcdc"));
+   //     openmct.install(GenericPlugin("miscellaneous"));
```
Comment (shihaocao) [inline MCT/public/index.html:41]: is this hardcoded? ie do we need to update this when we get a new control task?

File: cmdclient/__main__.py [hunk 2]
```diff
@@ -80,7 +80,9 @@ def do_add(self, args):
 
         else:
             endpoint = self.url+"/send-telem"
-            r = requests.post(url = endpoint, data = parseStateFields(args))
+            print(endpoint)
+            r = requests.post(url = endpoint, json = parseStateFields(args))
+            print(parseStateFields(args))
             print(r.content)
```
Comment (shihaocao) [inline cmdclient/__main__.py:86]: what does this do? Are these print statements debug statements or informative output?

File: cmdclient/config.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "FlightControllerFollower" : {
-        "type" : "RadioSession",
-        "server" : "",
-        "port" : 
+        "type" : "USBSession",
+        "server" : "localhost",
+        "port" : 8000
```
Comment (shihaocao) [inline cmdclient/config.json:5]: are these required changes? Should we have instead duplicated this config file?

File: ptest/http_cmd.py [hunk 4]
```diff
@@ -148,27 +148,31 @@ def create_usb_session_endpoint(usb_session):
...
+            # Get rid of uplink files/cleanup
+            os.remove("uplink.sbd")
+            os.remove("uplink.json")
 
+            if success:
+                return "Successfully sent telemetry to State Session"
+            return "Unable to send telemetry"
+        except:
+            import traceback
+            return traceback.format_exc()
```
Comment (shihaocao) [inline ptest/http_cmd.py:176]: Should this be in master? What's going on here?

File: MCT/public/index.html (hunk 1 above)
Comment (Duncan-McD) [reply MCT/public/index.html:41]: yes

**Review by shihaocao — APPROVED** [review]

File: cmdclient/__main__.py (hunk 2 above)
Comment (Duncan-McD) [reply cmdclient/__main__.py]: these changes were reverted because they weren't needed

File: cmdclient/config.json (hunk 3 above)
Comment (Duncan-McD) [reply cmdclient/config.json]: I reverted these changes as they were not needed

File: ptest/http_cmd.py (hunk 4 above)
Comment (Duncan-McD) [reply ptest/http_cmd.py]: I reverted these changes as they are not needed

**Comment by shihaocao** [comment]
Force merged because MCT changes only. and ci did not re-run

**Pull request merged**